R/prep_data.R

```diff
@@ -250,9 +254,21 @@ prep_data <- function(d,
     # recipe <- recipe %>% step_hcai_mostly_missing_to_factor()  # nolint
 
     # Remove columns with near zero variance ----------------------------------
+    options <- list(freq_cut = 95 / 5, unique_cut = 10)
+    if (!is.logical(remove_near_zero_variance)) {
+      if (!is.list(remove_near_zero_variance))
+        stop("remove_near_zero_variance must be logical or list for step_nzv")
+      if (all(sort(names(remove_near_zero_variance)) !=
+              c("freq_cut", "unique_cut"))) {
+        stop("remove_near_zero_variance must be a named list with 'freq_cut'
+               and 'unique_cut'")
+      }
+      options <- remove_near_zero_variance
```

Comment:
Rename this variable, both to be more specific and because `options` is a base-R function, which can lead to confusing errors.